README.md

```diff
@@ -1 +1,57 @@
-# ExtendedBrowser
+ExtendedBrowser
+===============
+
+A simple web browser component with address and navigation bars based on WebView.
+
+<img src="https://www.dropbox.com/s/es7q2z3qqty473p/ExtendedBrowser-screenshot.jpg?dl=1" width="300px" style="width: 300px;"/>
+
+
+Download
+--------
+
+Add application gradle dependency:
+
+```groovy
+  compile 'com.robotsandpencils.extendedbrowser:extendedbrowser:0.0.3'
+```
+
+Also add the following to project's gradle file:
+
+```groovy
+allprojects {
+    repositories {
+        maven { url 'https://dl.bintray.com/androidrnp/maven/' }
+    }
+}
+```
+
+
+Usage
+-----
+
+``` xml
+<com.robotsandpencils.extendedbrowser.views.ExtendedBrowser
+    android:id="@+id/web_view_browser"
+    android:layout_width="match_parent"
+    adroid:layout_height="match_parent"
+    app:showAddressBar="true"
+    app:showNavigationBar="true"/>
+```
+
+
+License
+-------
+    Copyright 2016 Robots & Pencils, Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
what are the other license types available? what is the limitation of this license?